src/mmw/js/src/modeling/gwlfe/runoff/templates/table.html

```diff
@@ -1,13 +1,13 @@
-<table class="table custom-hover" data-toggle="table">
+<table class="table custom-hover" data-toggle="table" data-height="auto" data-show-footer="true">
     <thead>
         <tr>
             {% for columnName in columnNames %}
                 {% if loop.index0 == 0 %}
-                    <th data-sortable="true" data-field="date" data-sort-name="_date_data" data-sorter="window.monthSorter">
+                    <th data-sortable="true" data-field="date" data-sort-name="_date_data" data-sorter="window.monthSorter" data-footer-formatter="Total">
                         {{ columnName }}
                     </th>
                 {% else %}
-                    <th data-sortable="true" data-sorter="window.numericSort">{{ columnName }} (cm)</th>
+                    <th data-sortable="true" data-sorter="window.numericSort" data-footer-formatter="window.sumFormatter" data-align="right">{{ columnName }} (cm)</th>
```

Comment:
It seemed easiest to use a `window.` function here, just like we do for `numericSort`. Not sure if there's better ways to do this.